ruff_lsp/settings.py

```diff
@@ -1,8 +1,19 @@
 from __future__ import annotations
 
+import enum
+
 from typing_extensions import Literal, TypedDict
 
-Run = Literal["onSave", "onType"]
+
+@enum.unique
+class Run(str, enum.Enum):
+    """When to run Ruff."""
+
+    OnType = "onType"
+    """Run Ruff on every keystroke."""
```

Comment:
Oh god, docstrings break my brain.... This is confusing after only having used languages where comments appear before the elements.

Reply:
Yeah, I agree but this is required for the editor to show the documentation :)